generator/index.js

```diff
@@ -39,6 +39,36 @@ module.exports = (api, opts, rootOpts) => {
       return src
     })
 
+    if(opts.useScss){
+      
+
+      //Modify App.vue (import bootstrap styles)
+      helpers.updateApp(src => {
+        let styleBlockIndex = src.findIndex(line => line.match(/^<style/))
+
+        if(styleBlockIndex === -1){ //no style block found
+          //create it with lang scss
+          src.push(`<style lang="scss">`)
+          src.push(`</style>`)
+
+          styleBlockIndex = src.length - 2
+        }
+        else{
+          //check if has the attr lang="scss"
+          if(!src[styleBlockIndex].includes('lang="scss')){
+            //if not, replace line with lang="scss"
+            src[styleBlockIndex] = '<style lang="scss">'
+          }
+        }
+
```

Comment:
Do we want to include the below import here? 
`  
@import '~bootstrap/scss/bootstrap.scss';
`

Referenced from:
https://bootstrap-vue.org/docs/reference/theming

Reply:
No, because the bootstrap build will be composed in `src/assets/scss/vendors/bootstrap-vue/index.scss`, so the user can import only the features he wants.
Maybe it can be entered as an option and let the user decide if he wants to use the whole css or just parts of it.